release.yaml

```diff
@@ -17,6 +17,9 @@ rancher-gatekeeper:
 rancher-gatekeeper-crd:
 - 100.1.0+up3.7.1
 rancher-istio:
```

Comment:
release.yaml should be deleted and recreated only containing these changes since it's an OOB release; all the other charts should be removed.

Reply:
Updated the release.yaml